packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -6,11 +6,162 @@
  * @prop --calcite-alert-width: Specifies the width of the component.
  */
 
+:host {
+  --calcite-alert-edge-distance: theme("spacing.8");
+  @apply block;
+}
+
+.container {
+  @apply bg-foreground-1
+  box-border
+  fixed
+  flex
+  flex-wrap
+  items-center
+  justify-center
+  mx-auto
+  my-0
+  opacity-0
+  pointer-events-none
+  shadow-2
+  text-start
+  w-full
+  z-toast;
+
+  container: responsive-container / inline-size;
+  border-radius: var(--calcite-border-radius);
+  border-block-start: 0 solid transparent;
+  border-inline: 1px solid var(--calcite-ui-border-3);
+  border-block-end: 1px solid var(--calcite-ui-border-3);
+  inline-size: var(--calcite-alert-width);
+  max-inline-size: calc(100% - (var(--calcite-alert-edge-distance) * 2 + 2px));
+  transition: var(--calcite-internal-animation-timing-slow) $easing-function,
+    opacity var(--calcite-internal-animation-timing-slow) $easing-function,
+    all var(--calcite-animation-timing) ease-in-out;
+
+  &--bottom,
+  &--top {
+    inset-inline-end: 0;
+    inset-inline-start: 0;
+  }
+  &[class*="bottom"] {
+    transform: translate3d(0, var(--calcite-alert-edge-distance), 0);
+    inset-block-end: var(--calcite-alert-edge-distance);
+  }
+  &[class*="top"] {
+    transform: translate3d(0, calc(-1 * var(--calcite-alert-edge-distance)), 0);
+    inset-block-start: var(--calcite-alert-edge-distance);
+  }
+  &[class*="start"] {
+    inset-inline-start: var(--calcite-alert-edge-distance);
+    inset-inline-end: auto;
+  }
+  &[class*="end"] {
+    inset-inline-end: var(--calcite-alert-edge-distance);
+    inset-inline-start: auto;
+  }
+}
+
+.content {
+  @apply box-border flex flex-auto flex-col transition-default;
+  padding-block: var(--calcite-alert-spacing-token-medium);
+  padding-inline: var(--calcite-alert-spacing-token-large) var(--calcite-alert-spacing-token-small);
+}
+
+.icon {
+  @apply flex flex-col items-center justify-start p-0;
+  margin-block-end: var(--calcite-alert-spacing-token-large);
+  margin-inline-end: auto;
+}
+
+.close {
+  @apply bg-transparent border-none cursor-pointer flex items-start justify-end outline-none self-stretch text-color-3;
+  -webkit-appearance: none;
+  padding: var(--calcite-alert-spacing-token-large);
+
+  @apply focus-base;
+  &:focus {
+    @apply focus-inset;
+  }
+
+  &:hover,
+  &:focus {
+    @apply bg-foreground-2 text-color-1;
+  }
+
+  &:active {
+    @apply bg-foreground-3;
+  }
+}
+
+.queue-count {
+  @apply text-color-2
+  bg-foreground-1
+  transition-default
+  invisible
+  flex
+  cursor-default
+  items-center
+  justify-around
+  self-stretch
+  overflow-hidden
+  text-center
+  font-medium
+  opacity-0;
+  border-inline: 0 solid transparent;
+  border-start-end-radius: 0;
+
+  &--active {
+    @apply visible opacity-100;
+  }
+}
+
+.dismiss-progress {
+  @apply absolute
+  block
+  w-full
+  overflow-hidden;
+  inset-inline: 0;
+  inset-block-start: -2px;
```

Comment:
For these 2px values maybe a sass variable or token would be nice

Reply:
Maybe we can do this in a follow-up PR. I'm not entirely sure if all 2px values represent the same thing. 🤔

Reply:
Yep sounds good 👍